### ISSUE TITLE
Client need to reconnect for MYSQL ACL Changes to become live

### ISSUE THREAD

**Issue by iloveautomation** [vernemq/vernemq#1804]
### Environment

- VerneMQ Version: 1,11.0
- OS: Ubuntu 20.04 LTS
- VerneMQ configuration (vernemq.conf) or the changes from the default
- Auth/ACL Using MYSQL
- Cluster size/standalone: 2 Nodes

### Expected behaviour

- ACL updated in MYSQL Table, Changes are present live. Client can access a new Topic (Subscribe/Publish) after updating ACL in DB

### Actual behaviour

- The client needs to reconnect for those changes to apply.

Is there any way to disable this Cache Layer? Maybe within the lua files?

**Comment by ioolkos** [comment]
> vmq-admin session reauthorize does not apply to ACL changes!

Ah, I see, `session reauthorize` only applies to the subscriptions, so it goes through the subscriptions and reauthorizes them. But not the Publish ACLs which is probably what you noticing.

**Comment by iloveautomation** [comment]
> > vmq-admin session reauthorize does not apply to ACL changes!
> 
> Ah, I see, `session reauthorize` only applies to the subscriptions, so it goes through the subscriptions and reauthorizes them. But not the Publish ACLs which is probably what you noticing.

Exactly. Is there any other way to update the Cache and reload the ACLs?
